### PR TITLE
get the max device channels appropriately

### PR DESCRIPTION
**Pull request by sharsonia**

### CODE REVIEW

**Comment by sharsonia** [comment]
Results from a test run with vm_size Standard_E32-8s_v3
[20210806-120643-140-validate_device_channels_change.log](https://github.com/microsoft/lisa/files/6946954/20210806-120643-140-validate_device_channels_change.log)

File: lisa/tools/ethtool.py [hunk 1]
```diff
@@ -300,6 +300,16 @@ def get_device_channels_info(
         )
 
         device_channel_info = DeviceChannel(interface, result.stdout)
+
+        # Find the vCPU count to accurately get max channels for the device.
+        result = self.node.execute(
```
Comment (squirrelsc) [inline lisa/tools/ethtool.py]: If there is no special reason, use `lscpu` to get cpu_count.

Comment (sharsonia) [reply lisa/tools/ethtool.py]: Done

**Comment by sharsonia** [comment]
Test result using lscpu command.

[20210807-025820-160-validate_device_channels_change.log](https://github.com/microsoft/lisa/files/6948611/20210807-025820-160-validate_device_channels_change.log)

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**